api/include/opentelemetry/trace/tracer.h

```diff
@@ -17,6 +17,7 @@ namespace trace
  * This class provides methods for manipulating the context, creating spans, and controlling spans'
  * lifecycles.
  */
+
```

Comment:
It looks like all the changes in this file are whitespace. Can you revert the file?

Reply:
I think that change is done by the formatting tools rather than me though. If I don't do that it will fail to pass the format test.

Reply:
I'm assuming that if you don't touch the file it doesn't need to be reformatted.

Reply:
Yeah but the formatter does seems to find this one and it is indeed that there should be a white space after the end of the file according to it. It's probably because of some legacy issues I suppose

Reply:
I don't think the formatter will complain if you return this file to it's current state in master. If it does, there's something wrong with the formatter, and that's an issue that can be addressed later on. I don't think this file should be changed in this PR, I recommend changing these whitespaces back. The same applies to all other files in this PR that only have whitespace changes.

When I'm formatting my files, I do Ctrl-Shift-i in VSCode on all of the files I've modified. I've never run into formatting issues with this approach. The problem here might be that you're running the formatter on the entire project, rather than only on files you're modifying.